crates/astria-sequencer/src/accounts/types.rs

```diff
@@ -89,6 +164,45 @@ impl std::ops::Sub<u128> for Balance {
     }
 }
 
+impl Balance {
+    pub fn as_proto(&self) -> ProtoBalance {
+        ProtoBalance {
+            hi: (self.0 >> 64) as u64,
```

Comment:
Should these make use of the functions on the primitive?

Reply:
yes! will update